README.md

```diff
@@ -71,3 +71,56 @@ Will respond like
     ]
 }
 ```
+# Mocks for testing Agency Self Sign Up flows
```

Comment:
This is really good documenting this info. Can we put this in a section for testing?
e.g.
## Testing
### Mocks
### Running tests
e.t.c

Reply:
The mock services i have set up locally are for developers to test it end to end.
Its not for unit testing and its not what a tester would require either.  Its for us to check the integration locally.

Therefore which section should it be in?